aten/src/ATen/core/NamedRegistrations.cpp

```diff
@@ -193,6 +193,12 @@ TORCH_LIBRARY_IMPL(aten, Named, m) {
   m.impl("hypot", CppFunction::makeFallthrough());
   m.impl("hypot.out", CppFunction::makeFallthrough());
   m.impl("hypot_", CppFunction::makeFallthrough());
+  m.impl("heaviside.Tensor", CppFunction::makeFallthrough());
```

Comment:
I wouldn't bother modifying this file, which is for named tensor support. That can be added later if you like.

Reply:
Thanks for the explanation, I'll delete this one for the time being.